### PR TITLE
Detail support channels

### PR DESCRIPTION
**Pull request by steren**
Put forward Stack Overflow as the main place for questions.

### CODE REVIEW

**Comment by steren** [comment]
Hold on, I'll also make changes to the nav bar

**Comment by steren** [comment]
@vladmos please take a look

**Comment by ittaiz** [comment]
Is this indeed going to be the policy? I think the drawback of SO is that
it takes time until one feels like a community exists around a tag and that
she will get answers.
In order for this to establish itself people will need to be attentive to
the tag and answer relatively quickly. At least IMHO.

**Comment by steren** [comment]
@ittaiz we have an internal rotation to monitor and answer Stack Overflow. Any new question should be processed with a similar SLO than on the mailing list.

I think we all agree that the mailing list is a pretty bad tool for Q&A and to create a knowledge base.

**Comment by ittaiz** [comment]
👍🏽
Sounds good.
I think I'd recommend to then add a short sentence clarifying the
difference between SO and the mailing list. What is the difference? What
should I send to the mailing list and not SO?

**Comment by steren** [comment]
I tried to do this in https://github.com/bazelbuild/bazel/pull/2623/files#diff-c4ba0e564305db1f9daac4aa6434dadf
I am happy to take suggestions

File: site/_includes/header.html [hunk 1]
```diff
@@ -36,7 +36,8 @@
               <a href="/blog">Blog</a>
             </li>
             <li><a href="https://github.com/bazelbuild/bazel">GitHub</a></li>
```
Comment (vladmos) [inline site/_includes/header.html]: Fontawesome has a Github icon, we can add it here as well: http://fontawesome.io/icon/github/

Comment (steren) [inline site/_includes/header.html]: It's important that GitHub appears more prominently than Twitter, Stack Overflow or Mailing list. When I tried to use an icon, it was less noticeable, thus I kept it as a plain word.

Do you think we should aim for using a maximum of icons?

**Comment by ittaiz** [comment]
@steren I think the big question is what should one "Discuss" in the mailing list? Your change just says "Ask questions on..." and then "Discuss on...". What should I discuss there?

**Comment by kchodorow** [comment]
@ittaiz I think we're encouraging anything that fits the [Stack Overflow guidelines](http://stackoverflow.com/help/on-topic) to be posted there.  

The list could be used for opinion-based questions ("why isn't the OS exposed in Skylark?") or announcements ("I wrote a new rule set for PiPy").

**Comment by ittaiz** [comment]
That sounds great. Maybe this should be expressed in this change.

**Comment by steren** [comment]
Yes I did not want to be too precise. I was thinking that just the fact to have two different terms "ask questions" and "discuss" would already better direct users to the right channel.

**Comment by ittaiz** [comment]
Maybe. For me they were confusing terms.

**Comment by steren** [comment]
Can this PRD be merged?

File: site/_includes/header.html (hunk 1 above)
Comment (vladmos) [inline site/_includes/header.html]: If it was on purpose then no, I just thought it was missed by mistake.

**Pull request closed without merging**